guides/common/assembly_using-external-databases.adoc

```diff
@@ -15,7 +15,7 @@ endif::[]
 To create and use external databases for {Project}, you must complete the following procedures:
 
 . xref:preparing-a-host-for-external-databases_{context}[].
-Prepare a {RHEL} 8 server to host the external databases.
+Prepare an {EL} server to host the external databases.
```

Comment:
You still missed the point I tried to make. This will expand to `an Red Hat Enterprise Linux server` for Satellite, which is grammatically incorrect.

Reply:
Sorry, I didn't pop out the stash.